src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/webdriver/JavaScriptUtils.kt

```diff
@@ -5,11 +5,10 @@ import org.openqa.selenium.WebDriver
 
 class JavaScriptUtils {
     companion object {
-
         @JvmStatic
-        fun <T> executeScript(driver: WebDriver, js: String): T {
+        fun <T> executeScript(driver: WebDriver, js: String, vararg args: Any): T {
             driver as JavascriptExecutor
-            return driver.executeScript(js) as T
+            return driver.executeScript(js, *args) as T
```

Comment:
Unchecked cast warning

Reply:
Hmmm... I don't think it would be possible in Java, is it possible in Kotlin?

Reply:
https://kotlinlang.org/docs/reference/typecasts.html#safe-nullable-cast-operator

Reply:
Oops, the safe cast does not work on generics due to type erasure. So we can remove generics by creating a method for each of:
`@return One of Boolean, Long, Double, String, List, Map or WebElement.`

If that's too much, then at least let's suppress it.

Reply:
The other thing is that I think we WANT to have an exception thrown if the JS returned a value of an unexpected type.

Reply:
suppressed.

Reply:
Yeah, that would be the part of transforming the `T?` local value into the `T` return value, ie.
```kotlin
val result = driver.executeScript(js) as? T
return result ?: throw Exception("Unexpected return type: ${result.javaClass}")
```
Which would basically reimplement the `ClassCastException`, so you're right, no point in that.
Let's just suppress it.